api/src/controllers/cards.controller.ts

```diff
@@ -122,6 +141,33 @@ export class CardsController {
 
     }
   } 
+  
+  public createCardRating = async (req: Request, res: Response, next: NextFunction) => {
```

Comment:
How is the "public" keyword used? I'm seeing it fairly frequently here but it isn't something I've come across in the courses yet.

Reply:
Great question, this is a concept related to Object Oriented Programming (OOP). In OOP classes are created, think of them as a group of values, functions, etc. In this case, we have a class called CardsController which contains many functions that are used elsewhere in the project. Because these functions are "public" we can access them in separate files. This might not be the best explanation but here is an article that goes into more depth and talks about other terms like "private" and "protected". [https://dev.to/bhagatparwinder/classes-in-js-public-private-and-protected-1lok#:~:text=The%20three%20major%20keywords%20at,class%20that%20instantiated%20the%20object.](url)